### PR TITLE
implement padEnd

### LINKED ISSUE #18: Implement String.prototype.padEnd
Implement a polyfill for String.prototype.padEnd as per https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padEnd#Polyfill

### PR DESCRIPTION
**Pull request by rixlabs**
Resolve #18 Implement String.prototype.padEnd

### CODE REVIEW

File: src/String.prototype.padEnd/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import padEnd from './padStart';
```
Comment (msn0) [inline src/String.prototype.padEnd/index.js]: should be 
```js
import padEnd from './padEnd';
```

**Comment by rixlabs** [comment]
should be ok! sry

**Comment by msn0** [comment]
Please squash commits into one and again the author, you know ;)

**Comment by rixlabs** [comment]
done sry :)

**Comment by msn0** [comment]
Thank you

**Pull request merged**